tests/sidetrail/DEBUGGING.md

```diff
@@ -0,0 +1,140 @@
+# Debugging SideTrail
+
+This is a guide for what to do when SideTrail reports a potential timing violation / throws a red `x` in CI.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+
+- [How to locally rerun the tests](#how-to-locally-rerun-the-tests)
+- [Updating the patch files](#updating-the-patch-files)
+- [Undefined functions](#undefined-functions)
+- [Loop related errors](#loop-related-errors)
+- [Warning: No entrypoints found.](#warning-no-entrypoints-found)
+- [SideTrail fails with a long error trace](#sidetrail-fails-with-a-long-error-trace)
+- [What to do if the proof gets really slow](#what-to-do-if-the-proof-gets-really-slow)
+- [Expected runtimes:](#expected-runtimes)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## How to locally rerun the tests
+The first step to debugging is to locally rerun the failing proof(s).
```

Comment:
minor nit, elsewhere you used a blank line between headers and body.  while not required, consistency is nice 🔢